ArcGISRuntimeSDKQt_CppSamples/Scenes/ViewPointCloudDataOffline/main.cpp

```diff
@@ -0,0 +1,69 @@
+// Copyright 2018 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifdef Q_OS_WIN
```

Comment:
This should also have `#include <QtCore/qglobal.h>` before this line, but I think all samples will need this. We can enter a new issue to follow up and do this consistently.